packages/appium/docs/en/intro/index.md

```diff
@@ -2,4 +2,200 @@
+to a kind of software module called an Appium *driver*. There's a whole [Driver
+Introduction](./drivers.md) which you can read next, so we won't go into huge detail on how they
+work for now.
+
+What's important to understand at the moment is that a driver is kind of like a pluggable module
+for Appium that gives Appium the power to automate a particular platform (or set of platforms,
+depending on the goal of the driver). At the end of the day, a driver's responsibility is to simply
+implement an Appium-internal interface representing the WebDriver protocol. How it implements this
+interface is totally up to the driver, based on its strategy for making automation happen on
+a specific platform. Typically, and with a lot more complexity and difficulty in the details,
+a driver does this by relying on platform-specific automation technologies. For example, Apple
+maintains an iOS automation technology called
+[XCUITest](https://developer.apple.com/documentation/xctest/user_interface_tests). The Appium
+driver that supports iOS app automation is called the [XCUITest
+Driver](https://github.com/appium/appium-xcuitest-driver) because ultimately what it does is
+convert the WebDriver protocol to XCUITest library calls.
+
+One of the reasons that drivers are independent, pluggable modules is that they work completely
+differently from one another. The tools and requirements for building and using drivers for
+different platforms are completely different. And so Appium lets you use just the drivers that you
+need for your automation tasks. Choosing drivers and installing them so that you can use them with
+your Appium instance is so important that Appium has its very own [CLI for managing
+drivers](../cli/extensions.md).
+
+So, to answer our original question, the way that Appium provides access to automation capabilities
+for a given platform is that the Appium team (or anyone else[^3]) writes a *driver* for that
+platform, implementing as much or little of the WebDriver protocol as desired. The driver can then
+be installed by anyone using Appium.
+
+[^3]: You can build and share your own drivers! Check out [Building
+  Drivers](../ecosystem/build-drivers.md) to learn more
+
+## Universal programming language access
+
+But what does it mean, or look like, to *use* Appium, anyway? Since Appium is ultimately a Node.js
+program, it *could* have looked like importing Appium and its drivers as libraries into your own
+Node.js programs. But that wouldn't meet Appium's goal of providing automation capabilities to
+people using any popular programming language.
+
+Luckily, the fact that Appium rode in on Selenium's coattails meant that we had a solution to this
+problem from day one. You see, the WebDriver specification is actually an HTTP-based protocol,
+meaning it is designed to be used over a network rather than within the memory of a single program.
+While there are some slight drawbacks in terms of performance and complexity to this model, the
```

Comment:
maybe we should avoid talking about the drawbacks because of the subjectivity of their severity 😄

Reply:
we could also just say "UI automation is so slow that adding HTTP over localhost on top is not going to make any difference" :-)